kivy/core/camera/__init__.py

```diff
@@ -118,7 +118,7 @@ def _update(self, dt):
         pass
 
     def _copy_to_gpu(self):
-        '''Copy the the buffer into the texture'''
+        '''Copy the buffer into the texture'''
```

Comment:
@Sumit112192  Add "." at the end of the sentence.

Reply:
Added